src/codegen/formatter/tests.rs

```diff
@@ -192,15 +192,19 @@ new(name:string):Car
 #[test]
 fn test() {
     let formatted_code = format_elz(
-        "//this is comment line
+        "module str
+module main
```

Comment:
there will not have two module declarations. Therefore, the test seems nonsense.